src/ui/qgisapp.ui

```diff
@@ -2258,6 +2260,27 @@ Click to select a geometry to offset.
 2nd click to specify offset distance.
 Ctrl+2nd click to offset a copy of the selected geometry.
 Ctrl+click with snapping to select another layer's geometry to copy and offset.
+</string>
+   </property>
+  </action>
+  <action name="mActionChamferFillet">
+   <property name="checkable">
+    <bool>true</bool>
+   </property>
+   <property name="icon">
+    <iconset resource="../../images/images.qrc">
+     <normaloff>:/images/themes/default/mActionChamferFillet.svg</normaloff>:/images/themes/default/mActionChamferFillet.svg</iconset>
+   </property>
+   <property name="text">
+    <string>Chamfer / Fillet</string>
+   </property>
+   <property name="toolTip">
+    <string>Chamfer / Fillet
+Creates a chamfer or fillet of a geometry:
+Click to select a geometry to operate.
+2nd click to specify the chamfer distance or fillet radius.
+Ctrl+2nd click to create a copy of the selected geometry.
+Ctrl+click with snapping to select another layer's geometry to copy and operate.
```

Comment:
Does the ctrl modifier still apply??

Reply:
see https://github.com/qgis/QGIS/pull/63527/files#diff-7ed5bdfb153b1620af72f740d8879928a6de308849cc8e043785c64d3d727b23